lang/rs/Cargo.toml

```diff
@@ -9,3 +9,7 @@ members = [
     "apis/cne/examples/loopback",
     "examples/echo_server"
 ]
+exclude = [
+    "examples/pktfwd"
+]
```

Comment:
pktfwd example is still earlier version and not using high level Rust APIs. So it's excluded from top-level cargo workspace. This will still work but it needs to be build independently from it's directory.  I'll raise a separate PR later to update pktfwd example also to use Rust CNE high level APIs.